oleesir/banking-app
# #164731195User should be able to see the create admin or staff user account page.

#### What does this PR do?
implement admin-bank-account-template
#### Description of Task to be completed?
implement admin-bank-account-template
#### How should this be manually tested?
* Navigate to the UI folder and click on staff_accounts.html in your browser
* You will see a create account button
#### What are the relevant pivotal tracker stories?
[ #164731195](https://www.pivotaltracker.com/story/show/164731195)
#### Any background context you want to add?
N/A
#### Screenshots
<img width="1280" alt="Screenshot 2019-03-29 at 8 00 38 PM" src="https://user-images.githubusercontent.com/17679511/55256548-8d71ea80-525d-11e9-99d6-7c57d2910143.png">
